src/components/ContactList.jsx

```diff
@@ -0,0 +1,40 @@
+import { useState } from 'react';
+import { Link } from 'react-router-dom';
+
+export default function ContactList(props)
+{
+    const { contacts } = props
+    const [searchTitle, setSearchTitle] = useState("")
+    let filteredContacts = contacts
+
+    if (searchTitle.length > 0)
+    {
+        filteredContacts = filteredContacts.filter((contact) => 
+        {
+            if (contact.firstName.toLowerCase().includes(searchTitle.toLowerCase())) return contact
```

Comment:
you are repeating the searchTitle.toLowerCase twice, maybe extract it out as a const variable